app/bolt_listeners.py

```diff
@@ -223,6 +239,23 @@ def respond_to_app_mention(
             )
 
 
+def append_image_content(files, content):
+    for file in files:
+        mime_type = file.get("mimetype")
+        if mime_type and mime_type.startswith("image"):
+            encoded_image, image_format = download_and_encode_image(file.get("url_private"), os.environ["SLACK_BOT_TOKEN"])
+            if image_format.lower() in ["jpeg", "png", "gif"]:
+                image_item = {
+                    "type": "image_url",
+                    "image_url": {
+                        "url": f"data:{mime_type};base64,{encoded_image}"
```

Comment:
can you add a comment referring to the relevant API document URL for future maintainers?

Reply:
added tp description